scripts/build/TestPlatform.Dependencies.props

```diff
@@ -57,7 +57,7 @@
     <!-- This version also needs to be updated in src\package\nuspec\TestPlatform.ObjectModel.nuspec -->
     <NuGetFrameworksVersion>5.11.0</NuGetFrameworksVersion>
     <ILAsmPackageVersion>5.0.0</ILAsmPackageVersion>
-    <JsonNetVersion>9.0.1</JsonNetVersion>
+    <JsonNetVersion>13.0.1</JsonNetVersion>
```

Comment:
Is there a reason to have a separate property here not use the on in Versions.props?

Reply:
I did not add it as you can see here. It was probably you who did not migrate it 😛